web/src/lib/domain-logic/isLicenseDataFromDatabaseDataMoreRecent.ts

```diff
@@ -0,0 +1,22 @@
+import { Business } from "@businessnjgovnavigator/shared/userData";
+import dayjs from "dayjs";
+
+export const isLicenseDataFromDatabaseDataMoreRecent = ({
+  businessFromDb,
+  businessFromUpdateQueue,
+}: {
+  businessFromDb: Business;
+  businessFromUpdateQueue: Business;
+}): boolean => {
+  if (typeof businessFromDb.licenseData !== typeof businessFromUpdateQueue.licenseData) return true;
```

Comment:
[dust/pebble] I'm a little confused on how different types indicate that one is more recent than the other?

Reply:
license data is initially undefined, if license data is an object for one of the parameters and undefined for the other, then one is more recent.